File: app/src/main/java/com/example/multiapplicationintent/MainActivity.java
package com.example.multiapplicationintent;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button b,b1,b2,b3;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        b=findViewById(R.id.button);
        b1=findViewById(R.id.button1);
        b2=findViewById(R.id.button2);
        b3=findViewById(R.id.button3);

        b.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = getPackageManager().getLaunchIntentForPackage("com.example.listview");
                startActivity(i);
            }
        });

        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i1 = getPackageManager().getLaunchIntentForPackage("com.example.switchapplication");
                startActivity(i1);
            }
        });

        b2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i2 = getPackageManager().getLaunchIntentForPackage("com.example.seekbar");
                startActivity(i2);
            }
        });

        b3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i3 = getPackageManager().getLaunchIntentForPackage("com.example.radiobutton");
                startActivity(i3);
            }
        });
    }
}

File: app/src/main/java/com/example/multiapplicationintent/Main_Activity.kt
package com.example.multiapplicationintent

import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    var b: Button? = null
    var b1: Button? = null
    var b2: Button? = null
    var b3: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        b = findViewById(R.id.button)
        b1 = findViewById(R.id.button1)
        b2 = findViewById(R.id.button2)
        b3 = findViewById(R.id.button3)
        b.setOnClickListener(View.OnClickListener {
            val i = packageManager.getLaunchIntentForPackage("com.example.listview")
            startActivity(i)
        })
        b1.setOnClickListener(View.OnClickListener {
            val i1 = packageManager.getLaunchIntentForPackage("com.example.switchapplication")
            startActivity(i1)
        })
        b2.setOnClickListener(View.OnClickListener {
            val i2 = packageManager.getLaunchIntentForPackage("com.example.seekbar")
            startActivity(i2)
        })
        b3.setOnClickListener(View.OnClickListener {
            val i3 = packageManager.getLaunchIntentForPackage("com.example.radiobutton")
            startActivity(i3)
        })
    }
}